### PR TITLE
New Style for notifications

### PR DESCRIPTION
**Pull request by robertkowalski**
I need some help for adjusting the colors.

bildschirmfoto 2015-01-05 um 15 16 51
bildschirmfoto 2015-01-05 um 15 17 03
bildschirmfoto 2015-01-05 um 15 17 18

### CODE REVIEW

File: assets/less/mixins.less [hunk 1]
```diff
@@ -20,9 +20,9 @@
         transition-timing-function: cubic-bezier(@a, @b, @c, @d); /* custom */
 }
 
-.bottom-shadow-border() {
+.bottom-shadow-border(@height: 6px) {
```
Comment (benkeen) [inline assets/less/mixins.less]: That's a sweet trick.

**Comment by benkeen** [comment]
Hey @robertkowalski, I'll let Sean critique any of the aesthetics if there's anything there. Code-wise it looks great. Very nice. +1

**Comment by kxepal** [comment]
Looks really good, but...

1. Add Option image: the notification box is too tall. Close button isn't on the same line as the text.

2. Other screenshots: the notification box is too large for the text it holds. How about to make it 30% of the screen, show in the middle, drop close X button and just close them by single click in in any place?

**Comment by kxepal** [comment]
As about the colors: on Add Option image you have two different red colors. Shouldn't Create button be green btw?

On second screenshot: again, two different red colors.

On third one: text should be clean white without green influence - this makes it less readable.

**Comment by kxepal** [comment]
Also, not sure why in email notification there are different screenshots, but this one:

image

doesn't overlaps left sidebar.

**Comment by robertkowalski** [comment]
Hi @kxepal , the screenshots are outdated, when you fetch the latest changes you will see that the colors changed as I got some feedback from Sean for the Fauxton theme

**Comment by benkeen** [comment]
This looks *really* great, @robertkowalski. Very smooth, looks & works well on FF + Chrome. I like the new colours. Code looks great.

- should there be a notification after adding an item on the config page? Right now nothing appears, which seems surprising. 
- one super minor thing I noticed was that the mouseover colour for links on the green notification looks dark & makes it a little hard to read. See screenshot. 
screen shot 2015-01-15 at 12 24 06 pm

But even with the above, +1 for sure!

**Comment by robertkowalski** [comment]
hi ben! thanks for the feedback! :)

 - no there was no notification for configuration saved successfully before
 - i just disabled the hover effect for links in the notifications for now

**Comment by robertkowalski** [comment]
all green, merged as 20bb445

**Pull request closed without merging**